RJL.UIP.HW3.ConditionalsMethods/Program.cs

```diff
@@ -219,23 +232,30 @@ static string[] GetMultiplyTable(int countPeople)
             string[] tempStringMultiplyArray = new string[countPeople];
             for (int i = 0; i < countPeople; i++)
             {
-                bool compareArrayItem = false;
-                do
+               do
                 {
                     firstValueTable = randomizer.Next(1, 9);
                     secondValueTable = randomizer.Next(1, 9);
                     resultMultiply = firstValueTable * secondValueTable;
                     resultStringMultiplyArray[i] = firstValueTable + "*" + secondValueTable + "=" + resultMultiply;
                     tempStringMultiplyArray[i] = secondValueTable + "*" + firstValueTable + "=" + resultMultiply;
-
-                    for (int j = 0; j < countPeople; j++)
-                    {
-                        compareArrayItem = ((resultStringMultiplyArray[i] == resultStringMultiplyArray[j]) || (tempStringMultiplyArray[i] == resultStringMultiplyArray[j])) && i != j;
-                    }
                 }
-                while (compareArrayItem);
+                while (CheckEqualArray(resultStringMultiplyArray, tempStringMultiplyArray, resultStringMultiplyArray[i],i));
             }
             return resultStringMultiplyArray;
         }
+        static bool CheckEqualArray(string[] FirstArrayForCheck, string[] SecondArrayforCheck, string valueForCheck, int indexOfArray)
```

Comment:
1. methods which return bool have the same rule as just variables - name start with **is**
2. you don't check are arrays equal because to don't check the sequence
3. in your case name should be **IsArraysHaveSameElements** etc